run.py

```diff
@@ -8,19 +8,23 @@
 from library.evaluation import classification_metrics
 from library.examples.all_transformations import all_transformations
 
+from plugins import IntegrityChecker, WandbPlugin, PipelineAnalyser
+
 
 train_dataset, test_dataset = load_data("data/original", preprocess_config)
 
 pipeline = hate_speech_detection_pipeline()
 
+default_plugins = [PipelineAnalyser(), IntegrityChecker()]
+custom_plugins = [WandbPlugin()]
 
 runner = Runner(
     pipeline,
     data={"input": train_dataset[Const.input_col]},
     labels=train_dataset[Const.label_col],
     evaluators=classification_metrics,
     train=True,
-    # plugins=[],
+    plugins=(default_plugins + custom_plugins),
```

Comment:
would be great if we made some plugins ovligayory, not sure if I 'd want these to be passed on!

Reply:
That is why we have default_plugins! Those are obligatory.

Reply:
are they obligatory? I can leave them out, no?

Reply:
I 'd add the default plugins in the runner, if we can! so they can not be left out

Reply:
Should I import them on a level lower in runner?

Reply:
Sounds good, I'll do it.